docs/develop/script-buttons.md

```diff
@@ -0,0 +1,32 @@
+---
+title: 'Run Office Scripts with buttons'
+description: Add buttons to workbooks that control Office Scripts.
+ms.topic: overview
+ms.date: 02/25/2022
+ms.localizationpriority: medium
+---
+
+# Run Office Scripts with buttons
+
+Help your colleagues find and run your scripts by adding script buttons to a workbook.
+
+:::image type="content" source="../images/run-from-button.png" alt-text="A button in the worksheet that runs a script when clicked.":::
+
+## Create script buttons
+
+With any script, go to the **More options (…)** menu in either the script's Details page or the Code Editor's task pane and select **Add button**. This creates a button in the workbook that runs the associated script when selected. It also shares the script with the workbook, so everyone with write permissions to the workbook can use your helpful automation.
```

Comment:
Why is "Details page" capitalized?

Reply:
Internally, that the name of this page, but it's never really called that elsewhere. I'll lower-case it.
![image](https://user-images.githubusercontent.com/38896772/156066005-03245013-bb54-45ca-943b-ef00c3b5fa72.png)